internal/cli/plugin/first_class.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/google/go-github/v61/github"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/plugin"
+	"github.com/spf13/cobra"
+)
+
+const (
+	SourceType = "first-class"
```

Comment:
Better change this name to `FirstClassSourceType` or something similar, since you've created this in the module `plugin`, not in the module `first_class_plugin` or something similar.

Reply:
done